app/models/concerns/notify_concern.rb

```diff
@@ -0,0 +1,70 @@
+module NotifyConcern
```

Comment:
I discussed with @allenwq, maybe just directly create the activity? No need to include the common module.